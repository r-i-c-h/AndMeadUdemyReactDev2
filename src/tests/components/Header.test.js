import React from 'react';
import Header from '../../components/Header';
import {shallow} from 'enzyme';

// import ReactShallowRenderer from 'react-test-renderer/shallow';
// test('should render Header component aok', () => {
//   const renderer = new ReactShallowRenderer();
//   renderer.render(<Header />);
//   expect(renderer.getRenderOutput()).toMatchSnapshot();
// });

// Enzyme Examples: 
  // expect( wrapper.find('h1').length ).toBe(1);
  // expect( wrapper.find('h1').text() ).toBe('Expensifical');
  
test('should render Header component aok', () => {
  const wrapper =shallow(<Header/>);
  expect(wrapper).toMatchSnapshot();
});