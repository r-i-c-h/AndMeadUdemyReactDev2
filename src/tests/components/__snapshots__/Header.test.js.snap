// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header component aok 1`] = `
<header>
  <h1>
    Expensifical
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    Create Expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/edit"
  >
    Edit Expense
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/help"
  >
    Help
  </NavLink>
</header>
`;
